### DISCUSSION TITLE
Welcome to BreakInfinity.cs Discussions!

### DISCUSSION THREAD

**Discussion by razenpok** [razenpok/BreakInfinity.cs#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NateAyye** [comment]
Hello, I love the work that everyone has done with this. 
I have loved using it in my unity games for needing numbers that could possibly get to big.
I was working with the script for a good minute and found a way to convert the scientific method string into the US Standard for reading large numbers and I was wondering if it would be something youd like to add in?

**Comment by Patashu** [comment]
Is it something you wrote code already for? If so, I'd like to see it.

**Reply by NateAyye** [reply]
Yes actually, I was working on a game for myself and wanted to handle insanly large numbers but also wanted standard naming, It was not fun trying to figure out and their are special cases in the names that i havent gotten quite yet such as adding an a or s depending on if the current prefix ends with an i or e. but other than that it has worked quite well.

![Method](https://user-images.githubusercontent.com/111668194/208833646-b5d0a9af-d23e-4f92-9add-d65b514962a3.png)
![Preffixes](https://user-images.githubusercontent.com/111668194/208833650-9a62f245-07ed-4011-a7cb-851474c5893d.png)

**Reply by NateAyye** [reply]
also it could probably use some refactoring. I was a little burnt out once i figured this one out.
